src/components/Header.js

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+import styled from "styled-components";
+import Logo from "../icons/Logo.js";
+
+const Title = styled.header`
+  height: 80px;
+  width: 375px;
+  background-color: rgb(0, 0, 0);
+  margin: 0;
+  display: flex;
+  border-width: 2px;
+  border-style: solid;
+  border-image: linear-gradient(to right, rgba(37, 206, 197, 1), black) 50 15%;
+`;
+
+const Container = styled.div`
+  align-items: center;
```

Comment:
are you sure `align-items` and `flex-direction` has any effect? 
this `div` is not a flexbox.

I think you don't need this container at all. 
Instead, you could style the `Logo` and apply `margin` there:

```
const StyledLogo = styled(Logo)`
  margin: 15px;
`
```